src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java

```diff
@@ -133,6 +133,8 @@
 	private static final String EMPTY_STRING = "";
 	private static final String SUPER_USER_ACTION_REQUESTS = "superUserActionRequests";
 
+	private static final String NODE_NAME_HIERARCHY_REQUEST = "Hierarchy Request";
```

Comment:
Any reason to not use the existing constant on line148? (Or PropDevRoutingStateConstants:170?)

Reply:
Didn't see that one. Ending up using a method call to make the code more readable.